performant-software/Neatline
# Static image layer workaround and zoom improvement

### What does this PR do?
- Adds a frontend fallback for checking image dimensions in case this fails in the backend due to PHP configuration
- Sets a minimum resolution of 0.5 on static image layers so that their zooming extent is more reasonable

### What issues does it address?
Closes #399, closes #395 

### How to test
- At http://neatline-staging.herokuapp.com/admin/neatline, click "Create Exhibit" and paste an image URL into the "Image layer" field. Enter an arbitrary number into the "Zoom levels" field. After saving, view the exhibit and use the layer switcher in the upper right to view the image layer. Confirm it appears correctly and can be zoomed to the number of levels indicated. Go back to the exhibit settings and change the zoom levels to a higher number; the zoom control should now go between the same minimum and maximum resolution but with finer granularity i.e. more levels in between.
